modules/phase_field/src/ics/MultiSmoothSuperellipsoidIC.C

```diff
@@ -34,6 +34,8 @@ InputParameters validParams<MultiSmoothSuperellipsoidIC>()
   params.addParam<MooseEnum>("check_extremes", extreme_options, "Check all Superellipsoid extremes (center +- each semiaxis) for overlap, must have prevent_overlap set to True.");
   MooseEnum overlap_options("False True", "False");
   params.addParam<MooseEnum>("prevent_overlap", overlap_options, "Check all Superellipsoid centers for overlap with other Superellipsoids.");
+  MooseEnum vary_axes_independently_options("False True", "True");
+  params.addParam<MooseEnum>("vary_axes_independently", vary_axes_independently_options, "If true the length of each semiaxis is randomly chosen within the provided parameters, if false then one random number is generated and applied to all semiaxes.");
```

Comment:
Yes please.

On Wed, Jul 6, 2016, 11:02 AM Floyd <notifications@github.com> wrote:

> In modules/phase_field/src/ics/MultiSmoothSuperellipsoidIC.C
> <https://github.com/idaholab/moose/pull/7282#discussion_r69770130>:
>
> > @@ -34,6 +34,8 @@ InputParameters validParams<MultiSmoothSuperellipsoidIC>()
> >    params.addParam<MooseEnum>("check_extremes", extreme_options, "Check all Superellipsoid extremes (center +- each semiaxis) for overlap, must have prevent_overlap set to True.");
> >    MooseEnum overlap_options("False True", "False");
> >    params.addParam<MooseEnum>("prevent_overlap", overlap_options, "Check all Superellipsoid centers for overlap with other Superellipsoids.");
> > +  MooseEnum vary_axes_independently_options("False True", "True");
> > +  params.addParam<MooseEnum>("vary_axes_independently", vary_axes_independently_options, "If true the length of each semiaxis is randomly chosen within the provided parameters, if false then one random number is generated and applied to all semiaxes.");
>
> Should i change the other two options i created in previous PR for this IC
> that made it through to merge but have the same problem?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/pull/7282/files/5c9688d3dc0c57ec73d16180ebc1c040284da57d#r69770130>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAMWPrFEOOViM6L7uE9DrmdM7sRoVuIjks5qS9-_gaJpZM4JGSPX>
> .
>